linkis-spring-cloud-services/linkis-service-gateway/linkis-spring-cloud-gateway/src/main/java/org/apache/linkis/gateway/springcloud/http/GatewayAuthorizationFilter.java

```diff
@@ -159,10 +159,14 @@ private Mono<Void> gatewayDeal(
     ServiceInstance serviceInstance;
     try {
       parser.parse(gatewayContext);
+      logger.info("client request uri: " + gatewayContext.getRequest().getRequestURI());
       if (gatewayHttpResponse.isCommitted()) {
         return gatewayHttpResponse.getResponseMono();
       }
       serviceInstance = router.route(gatewayContext);
+      if (serviceInstance != null) {
+        logger.info("route uri: " + serviceInstance.getInstance());
+      }
```

Comment:
The route method is already printed and contains more information, I think it can be removed

Reply:
I put the internal log on the outer layer so that it is easy to locate the problem in multiple threads.